integration-tests/src/test/java/org/zowe/apiml/functional/apicatalog/ApiCatalogStandaloneTest.java

```diff
@@ -0,0 +1,128 @@
+    private static final String GET_API_CATALOG_API_DOC_ENDPOINT = "/apicatalog/apidoc/service2/org.zowe v2.0.0";
+
+    private final static String USERNAME = ConfigReader.environmentConfiguration().getAuxiliaryUserList().getCredentials("servicesinfo-authorized").get(0).getUser();
+    private final static String PASSWORD = ConfigReader.environmentConfiguration().getAuxiliaryUserList().getCredentials("servicesinfo-authorized").get(0).getPassword();
+
+    private String baseHost;
+
+    @BeforeAll
+    static void init() throws Exception {
+        RestAssured.useRelaxedHTTPSValidation();
+    }
+
+    @BeforeEach
+    void setUp() {
+        ApiCatalogServiceConfiguration configuration = ConfigReader.environmentConfiguration().getApiCatalogStandaloneConfiguration();
+        String host = configuration.getHost();
+        String scheme = configuration.getScheme();
+        int port = configuration.getPort();
+        baseHost = scheme + "://" + host + ":" + port;
+        RestAssured.config = RestAssured.config().sslConfig(SSLConfig.sslConfig());
+        RestAssured.useRelaxedHTTPSValidation();
+    }
+
+    @Nested
+    class Containers {
+
+        @Nested
+        class HasRegisteredServices {
+
+            @Test
+            void whenGetContainers() throws IOException {
+                final ValidatableResponse response = given()
+                                                        .when()
+                                                            .get(baseHost + GET_ALL_CONTAINERS_ENDPOINT)
+                                                        .then()
+                                                            .statusCode(is(SC_OK))
+                                                            .contentType("application/json");
+
+                List<Map<String, Object>> list = response.extract().jsonPath().getList("$.");
+                assertEquals(2, list.size());
+                assertEquals("Mocked Services from file", list.get(0).get("title"));
+                assertEquals("Another mocked Services from file", list.get(1).get("title"));
+            }
+        }
+
+        @Nested
+        class ApiDocIsAvailable {
+
+            @Test
+            void whenGetApiDocDefaultEndpoint() {
+                final ValidatableResponse response = given()
+                                                        .when()
+                                                            .get(baseHost + GET_API_CATALOG_API_DOC_DEFAULT_ENDPOINT)
+                                                        .then()
+                                                            .statusCode(is(SC_OK))
+                                                            .contentType("application/json");
+                assertEquals("Service 2 - v1 (default)", response.extract().jsonPath().get("info.title"));
+            }
+
+            @Test
+            void whenGetApiDocv2Endpoint() {
+                final ValidatableResponse response = given()
```

Comment:
Just minor detail ... you can start with `when()` and delete 'given()' when there is nothing to set.

Reply:
Thank you, that's right, pushed.